packages/code-du-travail-frontend/pages/question.js

```diff
@@ -16,19 +15,16 @@ const {
   publicRuntimeConfig: { API_URL }
 } = getConfig();
 
-const fetchQuestion = ({ slug }) =>
+const fetchQuestion = async ({ slug }) =>
```

Comment:
way adding manual `async` ?
The function doesn't use `await` and already returns a Promise